spec/lib/medical_records/bb_internal/client_spec.rb

```diff
@@ -6,20 +6,19 @@
 
 describe BBInternal::Client do
   before(:all) do
-    VCR.use_cassette 'mr_client/bb_internal/session' do
-      VCR.use_cassette 'mr_client/bb_internal/get_patient' do
-        @client ||= begin
-          client = BBInternal::Client.new(session: { user_id: '15176497' })
-          client.authenticate
-          client
-        end
+    # The "get_patient" cassette also contains the session auth call.
+    VCR.use_cassette 'mr_client/bb_internal/get_patient', record: :new_episodes do
```

Comment:
Can you remove `record: :new_episodes` please

Reply:
@stevenjcumming good catch! I removed it.